### PR TITLE
add browser support page to blocks docs

### PR DESCRIPTION
**Pull request by MagdalenaHenke**
[Clubhouse](https://app.clubhouse.io/blocks/story/44375/move-blocks-browser-support-to-blocks-docs)

[Blocks Target Browsers Doc](https://docs.google.com/document/d/1tZT31TDBeTQqT7T4iwho9CRcnAsnVtxZ2cJhMlwTdRs/edit#heading=h.nynendc2a55s)

Just adding a new page with browser support information to the docs.

Screenshot of the new page:
Screen Shot 2019-11-21 at 3 02 58 PM

It's not the prettiest, but progress-over-perfection.

### CODE REVIEW

File: packages/blocks-docs/src/code_examples/ModalActionExample.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ class ModalActionExample extends React.Component {
 
     this.state = {
       modalVisible: false,
-      modalType: ''
+      modalType: null
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/code_examples/ModalActionExample.js]: unrelated fix - because `''` doesn't trigger default props, we kept a console warning about this prop being a value that wasn't allowed.

File: packages/blocks-docs/src/css/page.styl [hunk 2]
```diff
@@ -48,15 +48,15 @@ div[role="group"][tabindex] {
   padding-left: 5%;
   padding-right: 5%;
   padding-bottom: 40px;
-  
+
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/css/page.styl]: my sublime cuts off extra whitespace on save...

File: packages/blocks-docs/src/css/page.styl [hunk 3]
```diff
@@ -109,34 +109,34 @@ div[role="group"][tabindex] {
   }
 }
 
-.writing-guidelines-list {
+.help-text-list {
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/css/page.styl]: reusing it on the brower support page, so renamed the class

File: packages/blocks-docs/src/data/navigation.json [hunk 4]
```diff
@@ -85,6 +85,10 @@
         {
           "name": "Writing guidelines",
           "href": "/help/writing"
+        },
+        {
+          "name": "Browser Support",
+          "href": "/help/browsersupport"
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/data/navigation.json]: one _could_ argue it should be browserSupport, but I followed the path of `radiobuttons` being all lower case all together.

File: packages/blocks-docs/src/pages/help/browsersupport.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import React from "react"
+
+import Layout from "../../components/layout"
+import Header from "../../components/header"
+
+const WritingGuidelinesPage = () => (
+  <Layout>
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/pages/help/browsersupport.js]: I'm bothered by none of these pages setting the document title correctly or anything (it's all just the same anywhere on the page). Not something to fix here but an a11y thing I noticed that we should fix at some point.

File: packages/blocks-docs/src/pages/help/browsersupport.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    <Header title="Browser Support"></Header>
+    <div className="l-flex-vertical doc-section">
+      <div className="help-text-container">
+        <h2 className="blx-h3" id="heading-current">
+          Current browser support
+        </h2>
+        <p className="blx-with-margin">
+          Currently, BlocksDS is only tested in and actively supports one
+          browser:
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/pages/help/browsersupport.js]: happy to change any copy you don't like

**Comment by MagdalenaHenke** [comment]
@judyhe - I don't even think this needs review, I'd just like a "yeah, this copy is ok / this copy should be different".

**Comment by judyhe** [comment]
👍
